py3status/module.py

```diff
@@ -310,8 +310,12 @@ def set_module_options(self, module):
         Set universal module options to be interpreted by i3bar
         https://i3wm.org/i3status/manpage.html#_universal_module_options
         """
+        # keep i3bar module options in i3s module options.
+        # keep py3status module options in py3s module options.
+        # keep i3-gaps module options in gaps module options.
         self.i3s_module_options = {}
         self.py3_module_options = {}
+        self.gaps_module_options = {}
```

Comment:
looking much better now 

`self.gaps_module_options`  I don't like this name `self.i3_gaps_module_options`?

Also can the comment be by the definition not all above everything.  I should not have to scan the code to see things - it should all be super clear.

As an aside - I think background is fine with classic `i3-bar` for some time now (border is there but has issues so not practical)

we still need the `_get_color(border)` fix and do you also want the alpha channel?

Reply:
I made variable names more clear enough to not warrant a comment.

Reply:
https://github.com/Airblader/i3#background--color

It should be already available with `i3bar -t`. The exception is that this wouldn't work with CSS color names and general colors unless you have a solution in mind for those particular names. We also could support `rgba()` in the future. I think users could be more likely to use color hex over `rgba()` so maybe we don't really need to add anything here.

Reply:
@tobes Ping... How goes the `_get_color()` fix since it's the only thing preventing this PR from getting merged... I'm also okay with forgoing alpha channel if it's not an easy fix. Thanks.